### PR TITLE
Knowledge graph

### PR DESCRIPTION
**Pull request by HaigeWang1**

### CODE REVIEW

**Comment by HaigeWang1** [comment]
i create a compare author and a compare title function, so the main area can be more clear.

**Comment by hansvrapi** [comment]
> author and a compare title

that looks good :)

File: source/parser.py [hunk 1]
```diff
@@ -242,51 +423,47 @@ def main():
...
+                    # extract metadata for each paper using CERMINE 
+                    cermine =  CermineFile(paper_path + '.cermine')
+                    print(k, p)
+
+                    #compare the title and give the final result
+                    final_title = compare_title(grobid, cermine, url_path)
+                    
+                    #merge author information 
+                    final_author_list = compare_author(grobid, cermine)
+                    final_author_list = grobid.authors
```
Comment (hansvrapi) [inline source/parser.py:466]: did you add this line because the results from the author comparison method were not as expected?

**Review by hansvrapi — APPROVED** [review]

**Comment by hansvrapi** [comment]
@spiderCodeMan  can you please resolve the issues and merge the branch :) the changes seemed fine to me

**Pull request merged**